test/helpers/exchangeHelpers.ts

```diff
@@ -93,3 +95,45 @@ export const deployZeroExExchangeWrapperContract = async (
 
   return zeroExExchangeWrapperContract;
 };
+
+export const deployKyberNetworkWrapperContract = async (
+  kyberNetworkProxyAddress: Address,
+  transferProxy: TransferProxyContract,
+  core: CoreContract,
+  provider: Provider,
+): Promise<KyberNetworkWrapperContract> => {
+  const web3 = new Web3(provider);
+
+  const truffleKyberNetworkWrapperContract = contract(KyberNetworkWrapper);
+  truffleKyberNetworkWrapperContract.setProvider(provider);
+  truffleKyberNetworkWrapperContract.setNetwork(50);
```

Comment:
Why set network to 50? Is that our chain default?

Reply:
Yes